### PR TITLE
fix(cli): DB pooling and obf fix in cli

### PR DESCRIPTION
**Pull request by AnkurBansalSF**
## Description

Fixes :
- OBF was not working in services created using cli.
- loopback-connector-kv-redis was not installed automatically.
- added connection pooling for postgres datasource.
- added ratelimiter and helmet to facades created using cli

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## Checklist:

- [x] Performed a self-review of my own code
- [x] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/cli/src/generators/microservice/index.ts [hunk 1]
```diff
@@ -337,7 +348,7 @@ export default class MicroserviceGenerator extends AppGenerator<MicroserviceOpti
       scripts[symlinkresolver] = symlinkresolver;
       scripts['resolve-links'] =
         'npm run symlink-resolver build ./node_modules/@local';
-      scripts['prestart'] = 'npm run rebuild && npm run openapi-spec';
+      scripts['prestart'] = 'npm run clean && npm run openapi-spec';
```
Comment (akshatdubeysf) [inline packages/cli/src/generators/microservice/index.ts:351]: Why is this changed?

File: packages/cli/src/generators/microservice/index.ts [hunk 2]
```diff
@@ -481,6 +498,11 @@ export default class MicroserviceGenerator extends AppGenerator<MicroserviceOpti
           name: 'Redis',
           fileName: 'redis',
         },
+        {
```
Comment (akshatdubeysf) [inline packages/cli/src/generators/microservice/index.ts:501]: Why not use same store for ratelimiting?

File: packages/cli/src/generators/microservice/templates/src/application.ts.ejs [hunk 3]
```diff
@@ -163,6 +100,47 @@ export class <%= project.applicationName %> extends BootMixin(RestApplication) {
...
+        maxAge: 31536000,
+        includeSubDomains: true,
+        preload: true,
+      },
+    });
+
+    this.bind(RateLimitSecurityBindings.CONFIG).to({
+      name: 'RateLimitStore',
+      max: parseInt(process.env.RATE_LIMIT_REQUEST_CAP ?? '100'),
+      keyGenerator: rateLimitKeyGen,
```
Comment (akshatdubeysf) [inline packages/cli/src/generators/microservice/templates/src/application.ts.ejs:131]: Should use a better keygen as this one has some issues with the request IP.

**Review by akshatdubeysf — CHANGES_REQUESTED** [review]

File: packages/cli/src/generators/microservice/index.ts (hunk 1 above)
Comment (AnkurBansalSF) [reply packages/cli/src/generators/microservice/index.ts:351]: "rebuild": "npm run clean && npm run build",
"preopenapi-spec": "npm run build",
build was executed twice, so I changed it to clean

Comment (akshatdubeysf) [reply packages/cli/src/generators/microservice/index.ts:351]: oh ok

File: packages/cli/src/generators/microservice/templates/src/application.ts.ejs (hunk 3 above)
Comment (AnkurBansalSF) [reply packages/cli/src/generators/microservice/templates/src/application.ts.ejs:132]: Will have to change it in core

File: packages/cli/src/generators/microservice/index.ts (hunk 2 above)
Comment (AnkurBansalSF) [reply packages/cli/src/generators/microservice/index.ts]: Done

**Review by akshatdubeysf — APPROVED** [review]

**Review by yeshamavani — APPROVED** [review]

**Pull request merged**